src/app/designs/page.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import Card from '@/Components/Card';
+import Image from 'next/image';
+
```

Comment:
In general, kindly try to keep the file names also a little more descriptive. There are going to be other pages probably in the design folder (Please correct me if Im wrong), and it would be confusing for the others to understand the structure. Maybe a simple name like DesignPage.tsx, DesignComponent.tsx etc.

Reply:
renamed it DesignCard